docs/proposals/diagnose-experience.md

```diff
@@ -52,7 +52,7 @@ Logs focus on providing technical details for in-depth diagnosing and troublesho
 ## Proposals for ORAS CLI
 
 - Deprecate the global flag `--verbose` and only remain `--debug` to avoid ambiguity. Based on the concept above, it is reasonable to continue using `--debug` to enable the output of `DEBUG` level logs in ORAS as it is in ORAS. Meanwhile, This change will make the diagnose experience much more straightforward and less breaking since only ORAS `pull/push/attach/discover` commands have verbose output.
-- The existing output of `--verbose` in several ORAS commands `pull/push/attach/discover` can still be reserved. Introducing an additional flag to these commands.
+- Introduce a new flag `--detail` to replace the existing global flag `--verbose` of commands like `pull`, `push`, `attach`, and `discover` for detailed output.
```

Comment:
How about `--long`?

Reply:
I don't think `--detail` is a good option, since it will cause similar confusion as `--verbose`. Questions like is it about detailed logs or about detailed other outputs? 

One option is output the `--verbose` content by default, then we do not need any flags for verbose output that are not logs. This also solves the problem that some commands have `--verbose` flags, some don't.

Reply:
Will `--debug` be a global flag, or just available for some commands?

Reply:
Instead of `--detail`, I think we can have `--quiet` for `pull`, `push`, and `attach`.

`discover` is different since the `--verbose` takes effect on the metadata output instead of the status output. In that case, we can just remove `--verbose` and introduce new values for `--format` such as `--format tree-full`. It is also a good time to revisit the outputs of `oras discover`.

Reply:
Agree with @shizhMSFT . Current verbose output in `pull`, `push`, `attach` is progress information but in `discover` it is metadata output. They are different cases.

Reply:
Updated this proposal accordingly